pipeline/data_page/get_all_data_availability.py

```diff
@@ -357,37 +358,103 @@ def get_paralogs_summary(tc, depmap_paralogs_taiga_id):
     return paralogs_summary
 
 
-def get_id(possible_id, id_key="dataset_id"):
+def get_long_reads_summary(gcloud_storage_client, depmap_long_reads_gcloud_loc):
+    """
+    Get a summary of the long reads data available for each cell line.
+
+    Args:
+        gcloud_storage_client (storage.Client): gcloud storage client
+        depmap_long_reads_gcloud_loc (dict): gcloud location of the long reads data
+
+    Returns:
+        pd.DataFrame: Indexed by ACHID or ModelID, with a column of True values that will be filterd against the Model table in future
+    """
+
+    print("getting long_reads_summary...")
+
+    bucket_name = depmap_long_reads_gcloud_loc["bucket_name"]
+    prefix = depmap_long_reads_gcloud_loc["prefix"]
+    file_names = depmap_long_reads_gcloud_loc["file_names"].replace(" ", "").split(",")
```

Comment:
This will work, but you actually only need to read one file. The model IDs should be consistent across all files because we either have all types of data for the model or no data for the model. I don't think we should have a mix of different models in different files.